app/models/setting/provisioning.rb

```diff
@@ -58,7 +58,7 @@ def self.default_settings
       self.set('libvirt_default_console_address', N_("The IP address that should be used for the console listen address when provisioning new virtual machines via Libvirt"), "0.0.0.0", N_('Libvirt default console address')),
       self.set('update_ip_from_built_request', N_("Foreman will update the host IP with the IP that made the built request"), false, N_('Update IP from built request')),
       self.set('use_shortname_for_vms', N_("Foreman will use the short hostname instead of the FQDN for creating new virtual machines"), false, N_('Use short name for VMs')),
-      self.set('dns_conflict_timeout', N_("Timeout for DNS conflict validation (in seconds)"), 3, N_('DNS conflict timeout')),
+      self.set('dns_conflict_timeout', N_("List of timeouts (in seconds) for DNS lookup attempts (dns_lookup macro and DNS record conflict validation)"), [5, 10, 15, 20], N_('DNS conflict timeout(s)')),
```

Comment:
Can we rename this to `dns_timeout`? It's not just for conflict checking.

Reply:
Aaaaaghhhhh then I cannot probably use migration because Rails will initialize this. Damn. Need to start over.

Reply:
I think it could be ok not to have a migration. This is a very different implementation and we can warn users about this in the upgrade notes.